src/System.Windows.Forms/src/System/Windows/Forms/ContainerControl.cs

```diff
@@ -1326,6 +1352,84 @@ private void ScrollActiveControlIntoView()
             }
         }
 
+        protected override void RescaleConstantsForDpi(int deviceDpiOld, int deviceDpiNew)
+        {
+            if (deviceDpiNew == deviceDpiOld)
+            {
+                return;
+            }
+
+            base.RescaleConstantsForDpi(deviceDpiOld, deviceDpiNew);
+
+            // Checking if font is inherited from parent and is not being scaled by Parent.
+            // need to scale explicitly with new scaled font. (ex scenario: Winforms designer in VS)
```

Comment:
We need to scale what?
```suggestion
            // Check if Font is inherited from Parent and is not being scaled by Parent (e.g. the designer in VS).
            // In this case we need to scale <scale what????> explicitly with new scaled font. 
```

Reply:
I'm struggling to make a connection between this comment and `if TryGetExplicitlySetFont(..) return`.

Reply:
If font is explicitly set on control, we would update property bag.  This is checking for that. If it was set to inherit, then we need to respond for parent Font change event.